controller/contact.js

```diff
@@ -22,16 +17,56 @@ const schema = Joi.object({
 });
 
 const get = async (req, res, next) => {
+  const { page, limit, filter } = req.query;
+  const parsedPage = parseInt(page);
+  const parsedLimit = parseInt(limit);
+  const parsedFilter = filter === "true";
+
   try {
     const contacts = await service.getContacts();
```

Comment:
Nie pobieraj wszystkich tutaj, tylko wykorzystaj mongoose  opcje takie jak skip i limit:
    const result = await Contact.find(
      { owner, ...filter },
      '',
      { skip, limit }
    )

Reply:
To wprowadzi potrzebną Ci paginację.

Reply:
Limit ustaw sobie np na 20, a potem skip musisz obliczyć

Reply:
Udało się, poprzez Mongoose